packages/ng/core-select/input/select-input.component.ts

```diff
@@ -252,6 +267,7 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 
 	setDisabledState(isDisabled: boolean): void {
 		this.disabled$.next(isDisabled);
+		this.filterPillDisabled.set(isDisabled);
```

Comment:
could we just `toSignal(this.disabled$)`?

Reply:
Indeed !